src/omv/alloc/fb_alloc.h

```diff
@@ -33,7 +33,7 @@
 char *fb_alloc_stack_pointer();
 void fb_alloc_fail();
 void fb_alloc_init0();
-uint32_t fb_avail();
+uint32_t fb_avail(int hints);
```

Comment:
```suggestion
uint32_t fb_avail(int flags);
```

Also would like to rename those:
```c
enum {
    FB_ALLOC_FLAGS_FASTMEM = (1 << 0),
    FB_ALLOC_FLAGS_ALIGNED = (1 << 1),
} fb_alloc_flags_t;
```
If it's not fast, then it's size.

Reply:
Done